e2e-tests/steps/apply.ts

```diff
@@ -91,7 +91,7 @@ export const submitApplication = async (page: Page) => {
 
 export const viewSubmittedApplication = async (page: Page, name: string) => {
   await page.goto('/applications#submitted')
-  await expect(page.locator('h1')).toContainText('Short-Term Accommodation (CAS-2) applications')
+  await expect(page.locator('h1')).toContainText('Your CAS-2 applications')
```

Comment:
whoops, looks like I never merged this e2e test for viewing prison application https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-ui/pull/619 
do you want to just do the whole journey in this PR, starting from the dashboard instead of going directly to the seeded application?

Reply:
although I guess to ensure it's an application made by another user we will still need to use the seeded application...I wonder if there's a way to find a url with that ID on a page, in order to click it...

Reply:
hmmmm yeah, I remember this being tricky, but I'll have a look...

Reply:
if it's not possible we could have two e2e tests:
1. to check they can see a dashboard
2. to check they can visit  {seeded_application_id}/overview and leave a note